.github/workflows/coveralls.yml

```diff
@@ -1,12 +1,11 @@
 name: Code coverage
-
 on:
+    workflow_dispatch:
+    workflow_call:
+
     push:
         branches:
             - main
-    pull_request:
```

Comment:
Instead of running this separately, we're running this as part of the verification workflow and is called via workflow_call now instead.